### ISSUE TITLE
Indentation of the groups

### ISSUE THREAD

**Issue by Nic30** [gtkwave/gtkwave#50]
I do have nested groups in "Signals" view:
![gtkwave-groups](https://user-images.githubusercontent.com/5032806/111876262-3e6b7580-899e-11eb-9d9d-f7127ce6dee9.png)

Although the groups are nested they are displayed as they were of same level. If I click on them only a specific group highlights correctly so I guess that the groups are created correctly.

I expected something like this:
![image](https://user-images.githubusercontent.com/5032806/111876426-fac53b80-899e-11eb-94ff-85acde387dd6.png)

or 

![d3-wave](https://user-images.githubusercontent.com/5032806/111876435-031d7680-899f-11eb-93fe-4b4961e47012.png)

Is it possible to make groups somehow look like a tree in GTKWave?

**Comment by gtkwave** [comment]
I'll look into it. Thanks for the suggestion.

-Tony


     On Saturday, March 20, 2021, 12:10:17 PM EDT, Nic30 ***@***.***> wrote:  
 
 


I do have nested groups in "Signals" view:


Although the groups are nested they are displayed as they were of same level. If I click on them only a specific group highlights correctly so I guess that the groups are created correctly.

I expected something like this:


or



Is it possible to make groups somehow look like a tree in GTKWave?

**Comment by rishubn** [comment]
I agree, this would be very helpful, particularly with GHW file and VHDL record types

**Comment by Nic30** [comment]
@gtkwave  how long does it take from pull request to deb package?
Is there some "how to contribute" or I should just use common sense?

**Comment by xXIty** [comment]
Hello, has there been any update on this? @gtkwave

**Comment by gtkwave** [comment]
I've been far too busy with my day job so I haven't been able to work on this.
It's possible something similar will wind up in the gtk4 version as there has been talk of redoing how signals are handled as a tree emulated from a flat signal list, changed into an actual tree representation.  Please keep the feature request open as this could have traction for the future.

Thanks,
-Tony

**Comment by ColsonZhang** [comment]
So does the current gtk4 version supprt the function ?
